docs/model/datamodel.md

```diff
@@ -0,0 +1,198 @@
+---
+id: datamodel
+title: Data Model
+sidebar_label: Data Model
+---
+
+Graphback generates a GraphQL API and client-side documents using your data models. The data models are represented in the GraphQL Schema language as GraphQL types.
+
+In this section you will learn how to design and configure your data models for use with Graphback and how they can be used for multiple datasources.
+
+## Model
+
+Graphback processes GraphQL Schema types annotated with `@model`.
+Adding this annotation to your type will generate CRUD resolvers, services, data providers and client queries following the [GraphQLCRUD](https://graphqlcrud.org) format. For the full annotation definition see [`@model`](./annotations#model).
+
+```graphql
+"""
+@model
+"""
+type Note {
+  ...
+}
+```
+
+## Primary key
+
+Graphback requires each data model to have one primary key field which is used to uniquely represent every object in the database.
+
+### PostgreSQL
+
+For PostgreSQL, the default primary key format is `id: ID!`.
+
+```graphql
+"""
+@model
+"""
+type Note {
+  """Default primary key format for PostgreSQL"""
+  id: ID!
+  ...
+}
+```
+
```

Comment:
I think we are missing a little statement like `you can also use [@id](link to id annotation) annotation on field level to provide a custom primary key as shown below`